roles/swift-common/templates/etc/serverspec/swift_serverspec.rb

```diff
@@ -0,0 +1,66 @@
+require 'spec_helper'
+
+files = ['account-server.conf', 'container-server.conf', 'dispersion.conf', 'drive-audit.conf', 'object-expirer.conf', 'object-server.conf', 'proxy-server.conf', 'swift.conf']
+files.each do |file|  
+  describe file("/etc/swift/#{file}") do 
+    it { should be_owned_by 'swift' }
+    it { should be_grouped_into 'swift' }
+    it { should be_mode 640 }
+  end      
+end
+
+describe file('/etc/swift/') do
+  it { should be_owned_by 'swiftops' }
+  it { should be_grouped_into 'swiftops' }
+  it { should be_mode 755 } 
+end 
+
+files = ['account.log', 'container.log', 'object.log']
+files.each do |file|  
+  describe file("/var/log/swift/#{file}") do 
+    it { should be_owned_by 'syslog' }
+    it { should be_grouped_into 'adm' }
+    it { should be_mode 640 }
+  end     
+end
+
+describe file('/var/log/syslog') do 
+    it { should be_owned_by 'syslog' }
+    it { should be_grouped_into 'adm' }
+    it { should be_mode 640 }   
+end
+
+describe file('/dev/log') do 
```

Comment:
This and the syslog file above seem pretty generic, and not related to swift. Why are they here?